osgar/drivers/usbcam.py

```diff
@@ -0,0 +1,47 @@
+"""
+  Publish jpg images captured using libuvc (pupil-labs fork).
+  Only cameras directly supporting jpg output are supported.
+"""
+
+import uvc
+from threading import Thread
+
+from osgar.bus import BusShutdownException
+
+
+class UsbCam:
+    def __init__(self, config, bus):
+        self.input_thread = Thread(target=self.run_input, daemon=True)
+        self.bus = bus
+
+        bus = config.get('bus', 1)
```

Comment:
is bus=1 and port=0 default? if not I would use ['bus'] and ['port'] instead (i.e. the values have to be filled).
Maybe add here comment you have in PR, "how to find the bus/port number"??

Reply:
I considered `IndexError` exception not very user friendly but why not.